apps/style/common.scss

```diff
@@ -1,6 +1,6 @@
 @import "color";
 @import "font";
-@import "markdown";
+@import "curriculum/markdown";
```

Comment:
my only question is whether to move the markdown file into curriculum/. I think this seems fine -- doing this will imply that only curriculum-related markdown style rules belong in this file. if this stops being true, we can always hoist it back out.

Reply:
Ya I wondered on that too but for now it was just curriculum specific so figured it was better here.